src/store/tests/tests-reducer-factory.ts

```diff
@@ -22,6 +24,8 @@ export function testsReducerFactory(
   switch (category) {
     case TestCategory.ADI2:
       return testsCatADIPart2Reducer(action, state as Required<CatADI2UniqueTypes.TestResult>);
+    case TestCategory.B:
+      return testsCatBReducer(action, state as Required<CatBUniqueTypes.TestResult>)
```

Comment:
missing semi-colon, think this is an eslint error

Reply:
resolved